bin/hdfcoinc/pycbc_coinc_hdfinjfind

```diff
@@ -215,7 +218,7 @@ for trigger_file, injection_file in zip(args.trigger_files,
                  fo.create_group(key)
             fo[key].attrs['pivot'] = f[key].attrs['pivot']
             fo[key].attrs['fixed'] = f[key].attrs['fixed']
-            fo[key].attrs['foreground_time_exc'] = f[key].attrs['foreground_time_exc']
+        fo.attrs['foreground_time_exc'] = foreground_time_exc
```

Comment:
I think the output file should continue to have the individual `foreground_time_exc` fields for the different types of coinc time.  Otherwise we are throwing information from the statmap files away.